src/caffe/layers/cudnn_conv_layer.cu

```diff
@@ -61,19 +65,43 @@ void CuDNNConvolutionLayer<Dtype>::Backward_gpu(const vector<Blob<Dtype>*>& top,
   for (int i = 0; i < top.size(); ++i) {
     const Dtype* top_diff = top[i]->gpu_diff();
     // Backward through cuDNN in parallel over groups and gradients.
+#if CUDNN_VERSION_MIN(7, 0, 0)
+    for (int g = 0; g < 1; g++) {
+#else
     for (int g = 0; g < this->group_; g++) {
+#endif
       // Gradient w.r.t. bias.
       if (this->bias_term_ && this->param_propagate_down_[1]) {
-        CUDNN_CHECK(cudnnConvolutionBackwardBias(handle_[0*this->group_ + g],
+#if CUDNN_VERSION_MIN(7, 0, 0)
+        CUDNN_CHECK(cudnnConvolutionBackwardBias(handle_[g],
               cudnn::dataType<Dtype>::one,
               top_descs_[i],  top_diff + top_offset_ * g,
               cudnn::dataType<Dtype>::one,
               bias_desc_, bias_diff + bias_offset_ * g));
+#else
+        CUDNN_CHECK(cudnnConvolutionBackwardBias(handle_[g],
```

Comment:
Shouldn't we leave `CUDNN_CHECK(cudnnConvolutionBackwardBias(handle_[0*this->group_ + g],` here?